src/umpire/Replay.hpp

```diff
@@ -43,13 +46,13 @@ class Replay {
   static std::string printReplayAllocator(T&& firstArg, Args&&... args) {
     std::stringstream ss;
 
-    if (typeid(firstArg) != typeid(umpire::strategy::DynamicPool::CoalesceHeuristic)) {
+    //if (typeid(firstArg) != typeid(umpire::strategy::DynamicPool::CoalesceHeuristic)) {
       m_argument_number++;
       if ( m_argument_number != 1 )
         ss << ", ";
 
       ss << "\"" << firstArg << "\"";
-    }
+    //}
```

Comment:
I suggest we remove this 'if' since it seems to be working (or is it?, did we confirm that?)